lstchain/scripts/lstchain_find_pedestals.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""
+This script has to be launched (with no arguments) on a directory containing
+DL1 sub-run (i.e. not merged) LST real data files. It will find the
+event_id's of interleaved pedestals, and write them out into one hdf5 file for
+each sub-run, called pedestal_ids_RunXXXXX.YYYY.h5, which contains a single
+table with one column, 'event_id'
+"""
+
+import argparse
+import glob
+import os
+import numpy as np
+import pandas as pd
+
+from pathlib import Path
+from ctapipe.io import read_table
+from lstchain.paths import parse_dl1_filename
+
+parser = argparse.ArgumentParser(description="Interleaved Pedestal Finder")
+
+parser.add_argument(
```

Comment:
Make it an argument and don't provide a default. I think this should in the end be something like `<base>/monitoring/pedestal_ids/<date>/`

Reply:
It is an argument, do you mean "a required argument"?

Reply:
Sorry, unclear, positional argument, yes.